### PR TITLE
LPS-187148 Remove methods from DL Store api

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this about?

https://issues.liferay.com/browse/LPS-187148

This is a clean up, where I remove some methods from DL Store that are
not really needed.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/dxp/apps/antivirus/antivirus-async-store/src/main/java/com/liferay/antivirus/async/store/internal/AntivirusAsyncDLStore.java [hunk 1]
```diff
@@ -411,6 +394,29 @@ private void _registerCallback(DLStoreRequest dlStoreRequest)
...
+
+		_dlValidator.validateVersionLabel(versionLabel);
+
+		if (inputStream != null) {
+			_dlValidator.validateFileSize(
+				GroupThreadLocal.getGroupId(), fileName,
+				MimeTypesUtil.getContentType(fileName), inputStream);
+		}
+	}
+
```
Comment (marcogalluzzi) [inline modules/dxp/apps/antivirus/antivirus-async-store/src/main/java/com/liferay/antivirus/async/store/internal/AntivirusAsyncDLStore.java]: This method would raise an exception if parameter `sourceFilename` is not null and `fileExtension` is null, when calling `validateSourceFileExtension`. Maybe a fix is needed in `validateSourceFileExtension`?

**Review by marcogalluzzi — COMMENTED** [review]
Very good! Just one comment

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Review by marcogalluzzi — APPROVED** [review]

**Comment by austinchiang** [comment]
@adolfopa There is a startup error related to this pull. 

`bundle com.liferay.portal.reports.engine.console.web:5.0.42 (286)[com.liferay.portal.reports.engine.console.web.internal.display.portlet.DisplayPortlet(7023)] : findField: Suitable but non-accessible field _store in class com.liferay.portal.reports.engine.console.web.internal.admin.portlet.AdminPortlet, subclass of component class com.liferay.portal.reports.engine.console.web.internal.display.portlet.DisplayPortlet`

https://testray.liferay.com/home/-/testray/case_results/2352120271

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
Please @adolfopa check this error:
```
 [parallel] /opt/dev/projects/github/liferay-portal/portal-impl/src/com/liferay/portal/model/impl/ImageImpl.java:58: error: cannot find symbol 
 [parallel]           StringPool.BLANK);
 [parallel]           ^
 [parallel]   symbol:   variable StringPool
 [parallel]   location: class ImageImpl
```

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
ci:stable

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
ci:reevaluate:1397401_5577

**Comment by marcogalluzzi** [comment]
ci:reevaluate:1397401_5577

**Comment by marcogalluzzi** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:forward

**Comment by Hanlf** [comment]
The above failures are related to this PR.

**Pull request closed without merging**